### ISSUE TITLE
Sending two messages

### ISSUE THREAD

**Issue by MrDeluxe1** [jagrosh/Selfbot#74]
On some commands the bot will send 2 messages instead of just one.

**Comment by jagrosh** [comment]
You're running it twice

**Comment by MrDeluxe1** [comment]
I only have one open and it is still doing it.

**Comment by shikhir-arora** [comment]
Type `jps` Does this show more than one running?

**Comment by MrDeluxe1** [comment]
Where do I type jps?

**Comment by shikhir-arora** [comment]
I think it's unsupported for Windows, but the command prompt. Not a Windows user so I wouldn't be able to confirm that. I assume you are using Windows though.

**Comment by MrDeluxe1** [comment]
Yeah I'm on windows, could I look in the task manager and see it?

**Comment by shikhir-arora** [comment]
Yes you should be able to. Look for the running `.jar` file.

**Comment by MrDeluxe1** [comment]
I found the program, and there is only one running.

**Comment by jagrosh** [comment]
The only reason why it would respond twice is if you're running it twice. Go into Task Manager and close anything with 'java' in it, and then run the jar from the command line (_not_ from double-clicking it)

**Comment by MrDeluxe1** [comment]
Ok

**Comment by MrDeluxe1** [comment]
Hey! This fixed it thanks!
